core/src/main/java/hudson/model/Hudson.java

```diff
@@ -70,10 +69,10 @@
     @Deprecated
     private transient final CopyOnWriteList<ComputerListener> computerListeners = ExtensionListView.createCopyOnWriteList(ComputerListener.class);
 
-    /** @deprecated Here only for compatibility. Use {@link Jenkins#getInstance} instead. */
+    /** @deprecated Here only for compatibility. Use {@link Jenkins#get} instead. */
     @Deprecated
     @CLIResolver
-    @Nonnull
+    @Nullable
```

Comment:
WHy?  Heel no.  there is no JIRA and this will break the build of many plugins when upgrading the jenkins core